### PR TITLE
Fixes to address comments from PR 349

### PR DESCRIPTION
**Pull request by juazasan**
This PR fixes comments from ES team here: https://github.com/Azure/Enterprise-Scale/pull/394

### CODE REVIEW

File: docs/EnterpriseScale-Roadmap.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ We intend to update the content within this repo in alignment with Azure Semeste
...
 | Publish Enterprise-Scale GitHub Action in Actions marketplace | This enables Resource discovery, deployments and operationalize IaC. | Complete (June, 2020) |
 | Azure DevOps Support | Provide ability to operate AzOps within Azure DevOps with Azure Pipelines. | Complete (August, 2020) |
-| Additional reference implementations | Prescriptive first-party reference implementation for the Enterprise-Scale architecture for different enterprise scenarios and size | In progress (November, 2020) |
+| Additional reference implementations | Prescriptive first-party reference implementation for the Enterprise-Scale architecture for different enterprise scenarios and size | Complete (October, 2020) |
 | Workload Specific landing zones in Enterprise-Scale | AKS, WVD, SAP, HPC, Analytics <br/> (Seeking community Contribution) |  Planned |
 | Hybrid Management in Enterprise-Scale landing zones | Azure Arc | Planned |
+| Support for N regions | ES Reference Implementations (Contoso, Adventure Works) | January, 2021 |
+| Support for connecting N landing zones | ES Reference Implementations (Contoso, Adventure Works) | January, 2021 |
```
Comment (rjfmachado) [inline docs/EnterpriseScale-Roadmap.md:18]: @krnese @juazasan is there anything here that would prevent adding Trey Research to the list?

File: docs/reference/adventureworks/armTemplates/portal-es-hubspoke.json [hunk 2]
```diff
@@ -601,7 +601,7 @@
                     {
                         "name": "esAddressHub",
                         "type": "Microsoft.Common.TextBox",
-                        "label": "Address space (required for virtual network hub)",
+                        "label": "Address space (required for hub virtual hub)",
```
Comment (rjfmachado) [inline docs/reference/adventureworks/armTemplates/portal-es-hubspoke.json:604]: hub virtual hub, is this correct?

**Review by rjfmachado — APPROVED** [review]

File: docs/reference/adventureworks/armTemplates/portal-es-hubspoke.json (hunk 2 above)
Comment (juazasan) [reply docs/reference/adventureworks/armTemplates/portal-es-hubspoke.json:604]: Not related to Trey Research

File: docs/EnterpriseScale-Roadmap.md (hunk 1 above)
Comment (juazasan) [reply docs/EnterpriseScale-Roadmap.md:18]: Haven't seen that Adventure works already added support from connecting the Corp Landing Zones and which is also I also had in mind for Trey Research. I am not able to see any potential blockers to add similar capabilities in Trey Research. That said we agreed on closing this PR as it is now.

**Pull request merged**